code/style.css

```diff
@@ -1,58 +1,114 @@
 body {
-  margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  padding: 20px;
+  font-family: 'Vollkorn', serif;
 }
 
 p {
   margin: 1px;
   font-weight: 300;
-  background-color: #fff;
+}
+
+h1, h2, h3 {
+  font-family: 'Calligraffitti', cursive;
+  margin: 0px;
+
+}
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
 .cards {
-  margin: 50px auto;
-  width: 90%;
-  height: 600px;
-  background: #ccc;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  flex-direction: column;
+  width: 90%;
+  height: 950px;
+  background: #fff;
+  margin: 50px auto;
 }
 
-.killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+/* CARD DECKS */
+
+.killer-deck, .weapon-deck, .room-deck {
```

Comment:
I know this is a design thing, but it's important as a developer to think about these things as well. When you enter the  page the `color` on these decks are black. Black against the dark green is very hard to read, so maybe it will be better to use a more bright `color` for the text here